tests/sampling/test_mcmc_external.py

```diff
@@ -64,8 +68,11 @@ def test_external_nuts_sampler(recwarn, nuts_sampler):
     assert "L" in idata1.observed_data
     assert idata1.posterior.chain.size == 2
     assert idata1.posterior.draw.size == 500
+    assert idata1.posterior.tuning_steps == 500
     np.testing.assert_array_equal(idata1.posterior.x, idata2.posterior.x)
```

Comment:
@ricardoV94 `idata2` is used in this line only, to check if the sampling is deterministic I think?

Reply:
Ah okay! Thanks for checking